pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -281,7 +281,7 @@ func (c *cephStatusChecker) forceDeleteStuckRookPodsOnNotReadyNodes() error {
 
 func (c *cephStatusChecker) getRookPodsOnNode(node string) ([]v1.Pod, error) {
 	clusterName := c.clusterInfo.NamespacedName()
-	appLabels := []string{
+	rookAppLabels := []string{
```

Comment:
If you're renaming, how about this?
```suggestion
	rookLabels := []string{
```

Reply:
Or how about not renaming? Everything in rook is implicitly for rook.

Reply:
Not renaming sounds better